packages/server/graphql/public/mutations/startRecurrence.ts

```diff
@@ -2,22 +2,98 @@ import ms from 'ms'
 import {SubscriptionChannel} from 'parabol-client/types/constEnums'
 import {RRule} from 'rrule'
 import getRethink from '../../../database/rethinkDriver'
-import MeetingTeamPrompt from '../../../database/types/MeetingTeamPrompt'
 import {insertMeetingSeries as insertMeetingSeriesQuery} from '../../../postgres/queries/insertMeetingSeries'
+import restartMeetingSeries from '../../../postgres/queries/restartMeetingSeries'
+import {MeetingSeries} from '../../../postgres/types/MeetingSeries'
 import {analytics} from '../../../utils/analytics/analytics'
 import {getUserId, isTeamMember} from '../../../utils/authorization'
 import publish from '../../../utils/publish'
 import standardError from '../../../utils/standardError'
 import {MutationResolvers} from '../resolverTypes'
 
+const MEETING_DURATION_IN_MINUTES = 24 * 60 // 24 hours
+
+// Next meeting start is tomorrow at 9a UTC
+// :TODO: (jmtaber129): Determine this from meeting series configuration.
+const createNextMeetingStartDate = () => {
+  const now = new Date()
+  return new Date(Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate() + 1, 9))
+}
+
+const startNewMeetingSeries = async (viewerId: string, teamId: string, meetingId: string) => {
+  const now = new Date()
+  const r = await getRethink()
+  const nextMeetingStartDate = createNextMeetingStartDate()
+  const recurrenceRule = new RRule({
+    freq: RRule.WEEKLY,
+    byweekday: [RRule.MO, RRule.TU, RRule.WE, RRule.TH, RRule.FR],
+    dtstart: nextMeetingStartDate
+  })
+
+  const newMeetingSeriesParams = {
+    meetingType: 'teamPrompt',
+    title: 'Async Standup',
+    recurrenceRule: recurrenceRule.toString(),
+    duration: MEETING_DURATION_IN_MINUTES,
+    teamId,
+    facilitatorId: viewerId
+  } as const
+  const newMeetingSeriesId = await insertMeetingSeriesQuery(newMeetingSeriesParams)
+
+  await r
+    .table('NewMeeting')
+    .get(meetingId)
+    .update({
+      meetingSeriesId: newMeetingSeriesId,
+      scheduledEndTime: new Date(now.getTime() + ms('24h')) // 24 hours from now
+    })
+    .run()
+
+  return {
+    id: newMeetingSeriesId,
+    ...newMeetingSeriesParams
+  }
+}
+
+const restartExistingMeetingSeries = async (meetingSeries: MeetingSeries) => {
+  const r = await getRethink()
+  const {cancelledAt, recurrenceRule, id: meetingSeriesId} = meetingSeries
+  if (!cancelledAt) {
+    return
+  }
+
+  // to keep things simple, we'll just restart the meeting series at arbitrarty time, 9 AM tomorrow
```

Comment:
+1 Comment would be better `// to keep things simple, restart the meeting series at the same time a new series would start`

To future proof it we could even use the `currentRRule` to calculate which time it should start tomorrow
```ts
  const nextMeetingStartDate = currentRRule.after(new Date(Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate() + 1, 0)))
```

Reply:
good idea! fixed comment and added calculating date using `currentRRule`